addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -2139,6 +2140,56 @@ QUnit.test('chat window: chat header should not be clickable when thread has mul
     );
 });
 
+QUnit.test('2 chat windows: check shift operations have correct title with a LTR locale', async function (assert) {
+    assert.expect(2);
+
+    this.data['mail.channel'].records.push(
+        { is_minimized:true },
+        { is_minimized:true },
+    );
+    await this.start();
+
+    assert.strictEqual(
+        document.querySelector('.o_ChatWindowHeader_commandShiftPrev').title,
+        "Shift left",
+        "shift previous operation should be have 'Shift left' as title because locale is ltr"
+    );
+
+    assert.strictEqual(
+        document.querySelector('.o_ChatWindowHeader_commandShiftNext').title,
+        "Shift right",
+        "shift next operation should have 'Shift right' as title because locale is ltr"
+    );
+});
+
+QUnit.test('2 chat windows: check shift operations have correct title with a RTL locale', async function (assert) {
+    assert.expect(2);
+
+    this.data['mail.channel'].records.push(
+        { is_minimized:true },
+        { is_minimized:true },
+    );
+    core._t.database.parameters.direction = 'rtl';
+    await this.start({
+        env: {
+            _t: core._t,
+        }
+    });
+
+    assert.strictEqual(
+        document.querySelector('.o_ChatWindowHeader_commandShiftPrev').title,
+        "Shift right",
+        "shift previous operation should have 'Shift right' as title because locale is rtl"
+    );
+
+    assert.strictEqual(
+        document.querySelector('.o_ChatWindowHeader_commandShiftNext').title,
+        "Shift left",
+        "shift next operation should have 'Shift left' as title because locale is rtl"
+    );
+    core._t.database.parameters.direction = 'ltr';
```

Comment:
Why is it still here? I remember asking already but I don't see the thread on the line anymore.

Reply:
Because now it's changing the core value so we need to revert it back to what it was before the test changed it

Reply:
Wasn't it possible to just pass a proper mocked value for _t ?

Reply:
I tried many things to mock it but didn't succeed, as `t` is a function but with properties, I just didn't find a way to pass it to start function and make it work

Reply:
Something like this should work:

```js
Object.assign((s => s), {
    database: {
        parameters: {
            code: "en_US",
            date_format: '%m/%d/%Y',
            decimal_point: ".",
            direction: 'rtl',
            grouping: [],
            thousands_sep: ",",
            time_format: '%H:%M:%S',
        },
    },
});
```